Documentation/contributing/release/backports.rst

```diff
@@ -66,8 +66,10 @@ One-time setup
       $ git config --global user.email johndoe@example.com
 
 #. Make sure you have a GitHub developer access token with the ``public_repos``
-   scope available. For details, see `contrib/backporting/README.md
-   <https://github.com/cilium/cilium/blob/master/contrib/backporting/README.md>`_
+   scope available. You can do this directly from
+   https://github.com/settings/tokens or by opening GitHub and then navigating
+   to: User Profile -> Settings -> Developer Settings -> Personal access token
+    -> Generate new token.
```

Comment:
I think the extra space at the beginning of the line here is making the docs test fail.